core/flyout_button.ts

```diff
@@ -135,6 +140,7 @@ export class FlyoutButton
         },
         this.svgGroup!,
       );
+      aria.setRole(shadow, aria.Role.PRESENTATION);
```

Comment:
I hate that you have to do this but I don't know enough to suggest anything better. There is *very* little out there in terms of mark up for VO specifically.